onnx/backend/test/case/node/clip.py

```diff
@@ -16,70 +16,74 @@ class Clip(Base):
     def export():  # type: () -> None
         node = onnx.helper.make_node(
             'Clip',
-            inputs=['x'],
+            inputs=['x', 'min', 'max'],
             outputs=['y'],
-            min=-1.0,
-            max=1.0
         )
 
         x = np.array([-2, 0, 2]).astype(np.float32)
-        y = np.clip(x, -1, 1)  # expected output [-1., 0., 1.]
-        expect(node, inputs=[x], outputs=[y],
+        min_val = np.array(-1).astype(np.float32)
+        max_val = np.array(1).astype(np.float32)
+        y = np.clip(x, min_val, max_val)  # expected output [-1., 0., 1.]
+        expect(node, inputs=[x, min_val, max_val], outputs=[y],
                name='test_clip_example')
 
         x = np.random.randn(3, 4, 5).astype(np.float32)
-        y = np.clip(x, -1.0, 1.0)
-        expect(node, inputs=[x], outputs=[y],
+        y = np.clip(x, min_val, max_val)
```

Comment:
Could you add expected output in a comment (just like what you did right above)?

Reply:
good point for known inputs, bug how to add expected output for inputs that are randomly generated?